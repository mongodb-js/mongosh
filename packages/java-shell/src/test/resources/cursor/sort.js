// before
db.coll.remove({});
db.coll.insert({"_id": 1, name: "Vasya"});
db.coll.insert({"_id": 2, name: "Petya"});
db.coll.insert({"_id": 3, name: "Lyusya"});
// command
db.coll.find().sort({name: 1});
// clear
db.coll.drop();