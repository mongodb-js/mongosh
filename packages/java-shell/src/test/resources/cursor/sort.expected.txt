[ { "_id": 3, "name": "Lyusya" }, { "_id": 2, "name": "Petya" }, { "_id": 1, "name": "Vasya" } ]